master/docs/manual/configuration/workers-gce.rst

```diff
@@ -0,0 +1,105 @@
+.. -*- rst -*-
+
+.. index::
+   GCE
+   Workers; GCE
+
+.. bb:worker:: GCELatentWorker
+
+Google Compute Engine (GCE)
+=====================================================
+
+.. @cindex GCELatentWorker
+.. py:class:: buildbot.worker.gce.GCELatentWorker
+
+`GCE <https://cloud.google.com/compute/>`_ is a web service that allows you
+to start virtual machines in a Google data center. Please see their website
+for details, including costs.
+
+Using a GCE latent worker involves setting up a boot disk image and a GCE
+node. The worker will reset the node's disk using the "base" image and boot
+the virtual machine one time for each
+
+Compared to the Kubernetes latent worker, this allows you to use things
+that are currently not available on Kubernetes (e.g. OpenGL)
```

Comment:
I am pretty sure you can use the gfx with kubernetes
https://cloud.google.com/kubernetes-engine/docs/how-to/gpus

Reply:
Only EGL, not GLX.